tasks/security_agent.py

```diff
@@ -191,19 +191,20 @@ def run_functional_tests(ctx, testsuite, verbose=False, testflags='', fentry=Fal
 
 
 @task
-def run_ebpfless_functional_tests(ctx, cws_instrumentation, testsuite, verbose=False, testflags=''):
-    cmd = '{testsuite} -trace {verbose_opt} {testflags} {tests}'
+def run_ebpfless_functional_tests(ctx, testsuite, verbose=False, testflags=''):
+    if not testflags:
```

Comment:
Maybe we can check here if we only supplied `-test.v` and add it to the predefined list to add verbose without having to redefine the whole list?